prefect_dbt/cli/configs/bigquery.py

```diff
@@ -0,0 +1,96 @@
+"""Module containing models for BigQuery configs"""
+from typing import Any, Dict, Optional
+
+try:
+    from typing import Literal
+except ImportError:
+    from typing_extensions import Literal
+
+from prefect_dbt.cli.configs.base import MissingExtrasRequireError, TargetConfigs
+
+try:
+    from prefect_gcp.credentials import GcpCredentials
+except ModuleNotFoundError as e:
+    raise MissingExtrasRequireError("BigQuery") from e
+
+
+class BigQueryTargetConfigs(TargetConfigs):
```

Comment:
Can we add `_block_type_name`, `_logo_url`, `description`, and a code example to this block?